src/com/yetanalytics/lrs_admin_ui/views/special_actions.cljs

```diff
@@ -0,0 +1,8 @@
+(ns com.yetanalytics.lrs-admin-ui.views.special-actions
+  (:require [com.yetanalytics.lrs-admin-ui.views.delete-actor :refer [delete-actor]]))
+
+(defn special-actions []
+      [:div {:class "left-content-wrapper"}
+       [:h2 {:class "content-title"}
+        "Special Actions"]
```

Comment:
I'm not in love with this name but I guess that means I have to propose something better.

Reply:
also not in love with it, but it's v. easy to change.

Reply:
Statement Management?

Reply:
I like "Data Management."

Reply:
Now that I think about it, I should probably be updating the docs to make it very very very clear that DA is unrecoverable, do not pass go, you are probably looking for void.

Reply:
I won't submit a commit that changes the name until we're pretty sure.

Reply:
I'm good on Data Management. @kelvinqian00 ?

Reply:
I like it actually. It's consistent with "Credentials Management" and "Account Management."

My only concern, if you can call it that, is that having three "Management"s in the header would make it too long/crowded. I wonder if it's possible to abbreviate them to "Mgmt" if the browser window is narrowed, for example.

Reply:
I'm less worried than I _was_ about the crowdedness because the entire feature defaults to off

Reply:
Fair. We'll see how the aesthetics play out.